test/integration/Dockerfile

```diff
@@ -0,0 +1,22 @@
+FROM ubuntu:xenial
+
+# Minimal Docker container for running Habitat's BATS integration
+#  tests.
+#
+# jq and curl are used by our custom BATS helper functions; busybox
+# and the various lib* are needed by the hab binaries themselves; the
+# intended usecase is to exercise the individual binaries built by
+# cargo, and not Habitat-packaged binaries (which of course include
+# all dependencies).
+#
+# BATS, of course, is needed to run BATS :P
+
+RUN useradd hab
+
+RUN apt-get update && \
+    apt-get -y install jq curl git && \
+    apt-get -y install libarchive-dev libsodium-dev libczmq-dev busybox && \
```

Comment:
This should work well enough, although it depends which host you compiled on. Too bad `cargo build` didn't target static builds by default or this would have been tons easier.

Reply:
Yeah... I figure this is probably a decent first pass, though.

(I am once again reminded that I need to get a Windows VM set up 😦  )